From 8a6039df37353a3ef9208de0c4d63b611f549922 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Thu, 19 Oct 2017 12:46:53 -0400
Subject: [PATCH 1/2] *don't* use -f in dbxtool.service

Use -q to make it quiet but don't try to override it when we think it
won't work.

Related: rhbz#1489942

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 src/dbxtool.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/dbxtool.service b/src/dbxtool.service
index fcfb6e6..1a2a829 100644
--- a/src/dbxtool.service
+++ b/src/dbxtool.service
@@ -7,4 +7,4 @@ WantedBy=multi-user.target
 
 [Service]
 RemainAfterExit=yes
-ExecStart=/usr/bin/dbxtool -a /usr/share/dbxtool/ -q -f
+ExecStart=/usr/bin/dbxtool -a /usr/share/dbxtool/ -q
-- 
2.14.2

